### ISSUE TITLE
`git` interpreter command splits args unexpectidly

### ISSUE THREAD

**Issue by haunab-ornl** [olcf/pkpass#204]
When using the interpreter-exclusive `git` command, quoted args are split on spaces, instead of being preserved.

Expectation:
`git commit -am "chore: update example password"`
(expands to ["git" "commit" "-am" "chore: update example password"])
`[some_branch a1b2c3d] chore: Update Example Password`
`base/example/password +- 1`

Reality:
`git commit -am "chore: update example password"`
(expands to ["git" "commit" "-am" "chore:" "update" "example" "password"])
`fatal: paths 'update ...' with -a does not make sense`

**Comment by ginsburgnm** [comment]
yeah, this is a known problem I never got around to. I believe it's an issue with `Cmd` it's worth looking into a fix

**Comment by carljbai** [comment]
[shlex](https://docs.python.org/3/library/shlex.html) may work but it would need some thought.
